autogpt_platform/frontend/src/app/api/mutators/custom-mutator.ts

```diff
@@ -1,7 +1,26 @@
+import {
+  createRequestHeaders,
+  getServerAuthToken,
+} from "@/lib/autogpt-server-api/helpers";
+
 const FRONTEND_BASE_URL =
   process.env.NEXT_PUBLIC_FRONTEND_BASE_URL || "http://localhost:3000";
 const API_PROXY_BASE_URL = `${FRONTEND_BASE_URL}/api/proxy`; // Sending request via nextjs Server
 
+const isServerSide = (): boolean => {
```

Comment:
Can we make this shared utils? I see `window === "undefined"` used in a few places.